packages/sources/css/src/components/text-input/src/index.css

```
@@ -38,7 +38,8 @@ textarea.vtmn-text-input {
 }
 
 .vtmn-text-input:not(:disabled):not(:focus):not(textarea):hover {
-  box-shadow: inset 0 0 0 rem(1px) var(--vtmn-semantic-color_border-active);
+  box-shadow: inset 0 0 0 1px var(--vtmn-semantic-color_border-inactive),
+    0 0 0 3px var(--vtmn-semantic-color_hover-primary);
```

```suggestion
  box-shadow: inset 0 0 0 rem(1px) var(--vtmn-semantic-color_border-inactive),
    0 0 0 rem(3px) var(--vtmn-semantic-color_hover-primary);
```

Let's keep `rem()` function for the moment. This refactoring will be part of #526

---

I put back rem for every px size